spec/controllers/application_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.describe ApplicationController do
+  render_views
+  controller do
+    def index
+      raise ActionController::InvalidAuthenticityToken
+    end
+  end
+
+  describe "handling InvalidAuthenticityToken exceptions" do
+    it "renders the invalid session template" do
+      get :index
+      expect(response.body).to include(I18n.t!("sessions.invalid.heading"))
```

Comment:
Gonna be a bit nitpicky here. :)

If someone changes the value associated with `sessions.invalid.heading` to be something wrong, then this test would not pick up on it.

Also, it kind of knows too much about the internals of what it's testing. It would break if we ever decided to change the location of the text, e.g. `sessions.errors.invalid.heading`.